tools/pkglint/lib/rules.ts

```diff
@@ -1337,8 +1337,8 @@ export class PackageInJsiiPackageNoRuntimeDeps extends ValidationRule {
 }
 
 /**
- * Requires packages to have fast-fail build scripts, allowing to combine build, test and package in a single command.
- * This involves two targets: `build+test:pack` and `build+test` (to skip the pack).
+ * Requires packages to have fast-fail build scripts, allowing to combine build, test and package/extract in a single command.
+ * This involves multiple targets: `build+test`, `build+extract`, `build+test+extract`, and `build+test+package`
```

Comment:
Could you also update this file - https://github.com/aws/aws-cdk/blob/cd31457b1b75162cc1986e6e4410d72e663372c3/packages/%40aws-cdk/cfnspec/build-tools/create-missing-libraries.ts#L142

Would be nice to set the standard for new construct libraries to be rosetta enabled/supported.